components/buttons/ReplayButton.js

```diff
@@ -0,0 +1,17 @@
+import { Button } from "@chakra-ui/core";
+import { DARK_COLOR_100 } from "../../theme/colors";
+
+export function ReplayButton() {
+  return (
+    <Button
+      color={DARK_COLOR_100}
+      borderColor={DARK_COLOR_100}
+      fontSize="1.2em"
+      fontWeight="bold"
+    >
```

Comment:
You don't provide any props to the underlying button which means it does not receive the `onClick` prop you passed from the `Playboard` component

```suggestion
export function ReplayButton(props) {
  return (
    <Button
      color={DARK_COLOR_100}
      borderColor={DARK_COLOR_100}
      fontSize="1.2em"
      fontWeight="bold"
      {...props}
    >
```

Reply:
Ouch ! Thanks, I've fixed this component.